default.json

```diff
@@ -9,12 +9,12 @@
    "packageRules": [
       {
          "groupName": "all dependencies",
-         "matchPackageNames": [ ".*" ],
+         "matchPackageNames": [ "/.*/" ],
          "minimumReleaseAge": "7 days"
       },
       {
          "groupName": "silvermine packages",
-         "matchPackageNames": [ "^@silvermine/" ],
+         "matchPackageNames": [ "/@silvermine\//" ],
```

Comment:
I removed the carret because this pattern was not being captured in [regex101.com](https://regex101.com/?flavor=javascript&flags=ginst), which is the regex online runner recommended by them to test https://docs.renovatebot.com/string-pattern-matching/#use-regex101-to-test-your-patterns.